nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -1277,7 +1277,7 @@ struct nrf_wifi_event_raw_config_filter {
 	/** mode filter configured. */
 	unsigned char filter;
 	/** capture len configured. */
-	unsigned char capture_len;
+	unsigned int capture_len;
```

Comment:
Can you please map it to unsigned short to map it to upper layer mapping. It is uint16 for driver and above

Reply:
Done